README.md

```diff
@@ -3,6 +3,7 @@
 [![Version][jh-version-img]][jh-version-url]
 [![][docs-stable-img]][docs-stable-url]
 [![][gha-img]][gha-url]
+[![PkgEval][pkgeval-img]][pkgeval-url]
```

Comment:
```suggestion
[![pkgeval](https://juliahub.com/docs/JuliaHub/pkgeval.svg)](https://juliahub.com/ui/Packages/JuliaHub/B9bPq)
```
should we use the JuliaHub badge instead?

Reply:
https://github.com/JuliaComputing/JuliaHub.jl/pull/16/commits/57d4f272a410ae7ba49c6cc829ce62131b9774df